region/brinstar/red/Caterpillar Room.json

```diff
@@ -861,6 +958,48 @@
         "If the shinespark wind-up is activated as early as possible, then a Cacatac spike hit will be taken."
       ]
     },
+    {
+      "link": [2, 4],
+      "name": "Come in Shinecharging, Leave With Temporary Blue",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 7,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canChainTemporaryBlue",
+        {"or": [
+          "HiJump",
+          {"and": [
+            "canSpringBallJumpMidAir",
+            "canXRayTurnaround"
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}]
+    },
+    {
+      "link": [2, 4],
+      "name": "Come in Shinecharging, Leave With Temporary Blue (Spring Ball Jump)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 3,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canChainTemporaryBlue",
+        "canTrickySpringBallJump"
```

Comment:
I'm not sure I follow. Landing on the cac would just be unnecessary damage? And the bounce from the hit would probably put you too far right to make it up to the next ledge? I feel like I'm missing something; is there something you're wanting to change in the "requires" or is it about adding a note about the cac?

Updated the strat names to say "Full Runway" vs. "Short Runway" since the names may have been confusing before?

Reply:
How do you deal with the Cac?  Sometimes you can jump up alright, but the spines can be hard to avoid.  
If the cac step right, you can't kill it with blue and it becomes hard to shoot.
If it steps left you can kill it with blue.
Just doing the chainTemporaryBlue up to the door fast seems good enough.